### PR TITLE
Thanks for acception collaborate in your fork

### PR DESCRIPTION
**Pull request by double-beep**
The title
And an update from my repo

### CODE REVIEW

**Comment by double-beep** [comment]
Pls review and then I will mrg

**Comment by helper-beep** [comment]
Ok thanks for doing that for me.

File: trick.bat [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@echo off
+color 0a
+echo This is a trick!
+
+:TOP
+echo %random%
```
Comment (helper-beep) [inline trick.bat]: Very clever!!

Comment (helper-beep) [reply trick.bat]: @double-beep where did you learned that?

File: trick.bat [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@echo off
+color 0a
+echo This is a trick!
+
+:TOP
```
Comment (helper-beep) [inline trick.bat]: Goto  go to

**Comment by helper-beep** [comment]
First, unlocking conversation.

**Comment by helper-beep** [comment]
Closing, reopening and...

**Comment by helper-beep** [comment]
Merging!!!!!!!

**Pull request merged**